src/BcPrebidVast.js

```diff
@@ -670,22 +686,27 @@ function loadImaPlugin (callback) {
 		// add ima plugin script to the document body
 		var script = document.createElement('script');
 		script.src = 'https://players.brightcove.net/videojs-ima3/3/videojs.ima3.min.js';
+		script.id = 'bc_ima_plugin';
 		script.onerror = function (e) {
 			_logger.error(_prefix, 'Failed to load IMA Plugin. Error event: ', e);
+			_imaTriedToLoad = true;	// flag to load IMA plugin script only once
```

Comment:
I think this flag should be set when you first inject the <script> tag for IMA, not on the callback - otherwise there's a chance the script can be injected multiple times before the 1st one loads or errors out.

Reply:
After discussion, I understand that this logic makes sense. It's a little confusing, but it's workable.